smoot/criterion.py

```diff
@@ -0,0 +1,208 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Apr 26 10:26:43 2021
+
+@author: robin
+"""
+import numpy as np
+from scipy.stats import norm
+
+
+class criterion(object):
+    def __init__(self, name, models, ref=None, s=None):
+        self.modeles = models
+        self.name = name
+        self.ref = ref
+        self.s = s
+
+    def __call__(self, x):
```

Comment:
My next goal for this code is to modify my criterions' codes to adapt them to any number of objectives. I admit that you're proposition is cleaner than mine, but I prefer to stay with this disposition the time to be done with the modifications to not have too many .py docs to manage. Then when it will be done, I will do the change that you propose !